internal/commands/util/pr.go

```diff
@@ -41,6 +43,35 @@ func NewPRDecorationCommand(prWrapper wrappers.PRWrapper, policyWrapper wrappers
 	return cmd
 }
 
+func IsScanEnded(scansWrapper wrappers.ScansWrapper, scanID string) bool {
```

Comment:
in scan.go we have the function isScanRunning - This function also contains this logic.
Refactor Suggestion: Add to services scan.go file and implement the IsScanEnded in this file.
Then, make both PRDecorationGithub and isScanRunning to call the new scan service function

@OrShamirCM please add your opinion

Reply:
nice catch @AlvoBen !,if it's the same logic and there are no issues with the dependencies it's the right thing to do